INSTALL.md

```diff
@@ -1,108 +1,82 @@
-
-#### Linux
-To run PyBitmessage from the command-line, you must download the source, then
-run `src/bitmessagemain.py`.
+For Arch Linux
 ```
-git clone git://github.com/Bitmessage/PyBitmessage.git
-cd PyBitmessage/ && python src/bitmessagemain.py
+python2 openssl git python2-pyqt4 python-six
+```
+For Fedora
+```
+python python-qt4 git openssl-compat-bitcoin-libs python-six
+```
+For Red Hat Enterprise Linux (RHEL)
+```
+python python-qt4 git openssl-compat-bitcoin-libs python-six
+```
+For GNU Guix
+```
+python2-msgpack python2-pyqt@4.11.4 python2-sip openssl python-six
 ```
 
-That's it! *Honestly*!
-
-#### Windows
-On Windows you can download an executable for Bitmessage
-[here](https://github.com/Bitmessage/PyBitmessage/releases/download/0.6.3.2/Bitmessage_x86_0.6.3.2.exe).
+## setuptools
+This is now the recommended and in most cases the easiest procedure for installing PyBitmessage. 
 
-However, if you would like to run PyBitmessage via Python in Windows, you can
-go [here](https://bitmessage.org/wiki/Compiling_instructions#Windows) for
-information on how to do so.
+There are 3 options for running setuptools: root, user, venv
 
-#### OS X
-First off, install Homebrew.
+### as root:
 ```
-ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+python setup.py install
+pybitmessage
 ```
 
-Now, install the required dependencies
+### as user
+As normal user:
 ```
-brew install git python openssl cartr/qt4/pyqt@4 
+python setup.py install --user
+~/.local/bin/pybitmessage
 ```
 
-Download and run PyBitmessage:
+### as venv
+Create virtualenv with Python 2.x version
 ```
-git clone git://github.com/Bitmessage/PyBitmessage.git
-cd PyBitmessage && python src/bitmessagemain.py
+virtualenv -p python2 env
 ```
 
-## Creating a package for installation
-If you really want, you can make a package for PyBitmessage, which you may
-install yourself or distribute to friends. This isn't recommended, since
-PyBitmessage is in Beta, and subject to frequent change.
-
-#### Linux
-First off, since PyBitmessage uses something nifty called
-[packagemonkey](https://github.com/fuzzgun/packagemonkey), go ahead and get
-that installed. You may have to build it from source.
-
-Next, edit the generate.sh script to your liking.
-
-Now, run the appropriate script for the type of package you'd like to make
+Activate env
 ```
-arch.sh - create a package for Arch Linux
-debian.sh - create a package for Debian/Ubuntu
-ebuild.sh - create a package for Gentoo
-osx.sh - create a package for OS X
-puppy.sh - create a package for Puppy Linux
-rpm.sh - create a RPM package
-slack.sh - create a package for Slackware
+source env/bin/activate
 ```
 
-#### OS X
-Please refer to
-[this page](https://bitmessage.org/forum/index.php/topic,2761.0.html) on the
-forums for instructions on how to create a package on OS X.
-
-Please note that some versions of OS X don't work.
+Install requirements.txt
+```
+pip install -r requirements.txt
+```
```

Comment:
venv section is missing:
- how to build
- how to run

It only has how to install dependencies.